dagql/idtui/frontend_pretty.go

```diff
@@ -135,9 +135,6 @@ func (fe *frontendPretty) SetPrimary(spanID trace.SpanID) {
 }
 
 func (fe *frontendPretty) runWithTUI(ctx context.Context, ttyIn *os.File, ttyOut *os.File, run func(context.Context) error) error {
-	// NOTE: establish color cache before we start consuming stdin
-	out := NewOutput(ttyOut, termenv.WithProfile(fe.profile), termenv.WithColorCache(true))
```

Comment:
@vito do you remember what this is for? It doesn't seem to be essential from my testing, and it's only used in the `tea.WithOutput` call (but as of the new bubbletea dep, it expects the passed `io.Writer` to have an `Fd` method to consider it a file descriptor).